tdrs-frontend/src/assets/Reports.scss

```diff
@@ -58,4 +58,20 @@
 
```

Comment:
As discussed during office hours, my suggestion is to park this PR/issue. It is not high priority and is only impacting limited number of users (as far as I understand is only one user and that is not reproducible)